app/internal/page/app.go

```diff
@@ -515,14 +524,19 @@ func makeHandle(mux *http.ServeMux, logger Logger, store sessions.Store, localiz
 					return
 				}
 
-				sub, ok := session.Values["sub"].(string)
+				params, ok := session.Values["donor"].(*DonorLoginSession)
```

Comment:
I'm considering writing a helper for this, so we'd have `donorLoginSession, ok := getDonorSession(store)`, etc.

Reply:
and could probably return `nil` instead of `, ok`